docs/examples/params/model.yaml

```diff
@@ -0,0 +1,317 @@
+# Type of model that the rest of these parameters apply to.
```

Comment:
I'm not sure that we want to have the full documentation for all parameters in every model configuration file. Perhaps we could centralize this information somewhere and link to that reference page at the top of this file?